example/src/main/java/Extra.java

```diff
@@ -26,6 +27,7 @@ public static void show(final Application app) {
         ImGui.checkbox("Show ImGuiColorTextEdit Demo Window", SHOW_IMGUI_COLOR_TEXT_EDIT_WINDOW);
         ImGui.checkbox("Show ImGuiFileDialog Demo Window", SHOW_IMGUI_FILE_DIALOG_WINDOW);
         ImGui.checkbox("Show ImGui MemoryEditor Demo Window", SHOW_IMGUI_MEMORY_EDITOR_WINDOW);
+        ImGui.checkbox("Show ImGui Canva Demo Window", SHOW_IMGUI_CANVA_EDITOR_WINDOW);
```

Comment:
I believe we need to make the main window a little bit larger to remove a scrollbar.